openpower/package/hostboot/Config.in

```diff
@@ -25,7 +25,7 @@ config BR2_HOSTBOOT_CUSTOM_VERSION_VALUE
 
 config BR2_HOSTBOOT_VERSION
 	string
-	default "e5622fb032dc8b23627d8ca06d82a10b612d5435" if BR2_HOSTBOOT_LATEST_VERSION
+	default "ca03643dd8935f9c839cf8dfd4971c519476f4a1" if BR2_HOSTBOOT_LATEST_VERSION
```

Comment:
@dcrowell77 what's the right thing to have here? It merged, so I'm assuming the merge is an okay thing to do?

Reply:
@stewart-ibm -- Can this commit just not update hostboot? We have other PRs that will keep updating this as commits roll in (example here: https://github.com/ibm-op-release/op-build/pull/560).